Corvette rework.

Flaksim opened the pull request:
changes: 
*"Remodel of Corvette, second deck added."
*"Added functional air sensors to Corvette."
*"Made drop pods have a range of 3 tiles to enable flyby deployments."
*"Added emergency voidsuits to emergency lockers."

Flaksim committed:
work on umbilical tether

Flaksim committed:
work in progress corvette

Flaksim committed:
Merge branch 'alpha' of https://github.com/Flaksim/HaloSpaceStation13 into alpha

Flaksim committed:
Corvette Rework

changes:
*"Remodel of Corvette, second deck added."
*"Added functional air sensors to Corvette."
*"Made drop pods have a range of 3 tiles to enable flyby deployments."
*"Added emergency voidsuits to emergency lockers."

bloxgate commented:
Did you make sure that the ship's hitbox is still acceptable with the new deck and other changes?

Flaksim committed:
travis fix, hitbox adjustment.

Flaksim committed:
area fix travis.

Flaksim committed:
Blast door fix.

Flaksim committed:
Adds ONI Fatigues and Marine Medic Armor.

Flaksim committed:
rename of an item.

Flaksim committed:
Radical nerf of missile pods.

Changed fire delay from 2.5 seconds to 25 seconds to see if that is more balanced.

BDpuffy420 commented:
Just needs a change log and its good
